src/Services/Authentication/Authentication.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,100 @@
+using Authentication.BusinessLogic.DTOs.Request;
+using Authentication.BusinessLogic.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Shared.Constants;
+
+namespace Authentication.API.Controllers;
+
+
+[Route("api/[controller]")]
+[ApiController]
+public class UserController(IUserService _userService): ControllerBase
+{
+    [HttpPost("register")]
+    public async Task<IActionResult> RegisterAsync([FromBody] UserRequestDto model)
+    {
+        if (!ModelState.IsValid)
```

Comment:
Why are you doing this here? You have auto-validation if validation fails, the execution will not enter the controller